src/main/java/frc/robot/sensors/ArmSensor.java

```diff
@@ -46,23 +51,24 @@ public Distance getSensorDistance() {
   public Trigger inTrough() {
     return new Trigger(
         () -> {
-          if (getSensorDistance().in(Millimeters) > TROUGH_LOWER_LIMIT
-              && getSensorDistance().in(Millimeters) < TROUGH_UPPER_LIMIT) {
+          if (getSensorDistance().in(Meters) > TROUGH_LOWER_LIMIT
+              && getSensorDistance().in(Meters) < TROUGH_UPPER_LIMIT) {
             return true;
           } else {
             return false;
           }
         });
   }
 
-  public BooleanSupplier inClaw() {
-    return () -> {
-      if (getSensorDistance().in(Millimeters) > CLAW_LOWER_LIMIT
-          && getSensorDistance().in(Millimeters) < CLAW_UPPER_LIMIT) {
-        return true;
-      } else {
-        return false;
-      }
-    };
+  public Trigger inClaw() {
+    return new Trigger(
+        () -> {
+          if (getSensorDistance().in(Meters) > CLAW_LOWER_LIMIT
+              && getSensorDistance().in(Meters) < CLAW_UPPER_LIMIT) {
+            return true;
+          } else {
+            return false;
+          }
```

Comment:
You don't need to do quite this much.  Do similar to what you did for InPreIntakePostion().

`return new Trigger(
    () -> {
      var distance = getSensorDistance().in(Meters);
       return (distance > CLAW_LOWER_LIMT && distance < CLAW_UPPER_LIMIT);
   )
);`

Reply:
Do similar above.  No need to return true/false;